core/compiler.common.jvm/src/org/jetbrains/kotlin/load/java/JvmAnnotationNames.kt

```diff
@@ -36,7 +36,8 @@ val NULLABLE_ANNOTATIONS = listOf(
     FqName("edu.umd.cs.findbugs.annotations.CheckForNull"),
     FqName("edu.umd.cs.findbugs.annotations.Nullable"),
     FqName("edu.umd.cs.findbugs.annotations.PossiblyNull"),
-    FqName("io.reactivex.annotations.Nullable")
+    FqName("io.reactivex.annotations.Nullable"),
+    FqName("io.reactivex.rxjava3.annotations.Nullable")
```

Comment:
I'm not sure if it's possible to use trailing commas here? As in I'm not sure what kotlin language level the compiler code targets